src/lsst/cmservice/client/queues.py

```diff
@@ -115,13 +115,20 @@ def pause_until_next_check(
         row_id: int
             ID of the Queue row in question
         """
-        queue = self.get_row(row_id)
-        sleep_time = self.sleep_time(row_id)
-        wait_time = min(sleep_time, queue.interval)
-        delta_t = timedelta(seconds=wait_time)
-        next_check = queue.time_updated + delta_t
         now = datetime.now()
-        print(now, sleep_time, wait_time, next_check)
+        try:
+            queue = self.get_row(row_id)
+            sleep_time = self.sleep_time(row_id)
+            wait_time = min(sleep_time, queue.interval)
+            delta_t = timedelta(seconds=wait_time)
+            next_check = queue.time_updated + delta_t
+        except Exception as msg:  # pylint: disable=broad-exception-caught
+            print(msg)
```

Comment:
should these `print` statements be logs instead?

Reply:
Do we have a logger?